x-pack/plugins/cross_cluster_replication/public/plugin.ts

```diff
@@ -60,6 +63,12 @@ export class CrossClusterReplicationPlugin implements Plugin {
           history,
           getUrlForApp,
         });
+
+        return () => {
+          // Change tab label back to Kibana.
```

Comment:
FWIW I think this comment should be updated to "...back to Elastic" because that is the actual value. I think we could actually remove this comment entirely as I think the function name and object name are descriptive enough :)